test/fail_compilation/checkimports2c.d

```diff
@@ -2,9 +2,11 @@
 /*
 TEST_OUTPUT:
 ---
-fail_compilation/checkimports2c.d(20): Deprecation: local import search method found variable imports.imp2.X instead of variable checkimports2c.X
-fail_compilation/checkimports2c.d(29): Deprecation: local import search method found variable imports.imp2.X instead of variable checkimports2c.X
-fail_compilation/checkimports2c.d(30): Deprecation: local import search method found variable imports.imp2.Y instead of variable imports.imp1.Y
+fail_compilation/checkimports2c.d(22): Deprecation: local import search method found variable imports.imp2.X instead of variable checkimports2c.X
+fail_compilation/checkimports2c.d(28): Deprecation: local import search method found variable imports.imp2.X instead of nothing
+fail_compilation/checkimports2c.d(29): Deprecation: local import search method found variable imports.imp2.Y instead of nothing
```

Comment:
Looking at the test case, those don't seem very clear.
Could we have something that isn't refering to the compiler internal, e.g:
`Deprecation: Variable imports.imp2.Y introduced by a private local import is not visible with the new import rule, use a public import or an alias` ?

Other suggestions welcome, but "found X instead of nothing" is not user friendly.